web/src/components/views/Transform.vue

```diff
@@ -55,17 +60,38 @@ export default {
     txTypeOrNull(tx) {
       if (tx && 'transform_type' in tx) return tx.transform_type;
       return null;
-    }
+    },
+    async loadPCAData (csv) {
+      const pcaData = await CSVService.getPlot(csv, 'pca');
+      this.points = pcaData.data;
+    },
+    async loadPCADataShift () {
```

Comment:
Where is this function called?

Reply:
No, this was just a testing function I was using to simulate updates. Removed in e0b0e4c.